data_source_test.go

```diff
@@ -190,7 +190,8 @@ func TestWritingFiles(t *testing.T) {
 	if ds.processors[0].DataPublisher.HasLJH3() {
 		t.Error("Stop did not result in !HasLJH3")
 	}
-	if true { // prevent variable from persisting
+
+	if true { // block prevents variable from persisting (what variable?)
```

Comment:
maybe `err2`?

Reply:
Well, I don't see it. I wonder if it was originally my comment? Nope, it was you, on October 18, 2018: https://github.com/usnistgov/dastard/blame/98232542e6d78fe689adb21def07cf8208e1011b/data_source_test.go#L193

Reply:
My guess is the comment is no longer accurate? I'll remove the block, assuming it still passes tests.

Reply:
I mean, I'll keep the _contents_ of the block, but remove the nesting into a pointless block.